src/index.js

```diff
@@ -271,11 +273,12 @@ module.exports.warmUp = async (event, context, callback) => {
   console.log("Warm Up Start");
   const invokes = await Promise.all(functionNames.map(async (functionName) => {
     const params = {
+      ClientContext: "${Buffer.from(`{"custom":${this.payload}}`).toString('base64')}",
```

Comment:
Is there any reason for this to have the custom field instead of simply be: `      ClientContext: "${Buffer.from(this.payload).toString('base64')}",`